tests/SDKTest.php

```diff
@@ -287,9 +288,20 @@ public function testHTTPSuccess()
                 
                 $this->assertEquals('GET:/v1/mock/tests/general/redirect/done:passed', $output[10]);
                 
-                if($options['supportUpload']) {
+                $supportsUpload = $options['supportUpload'];
```

Comment:
Unless we don't use this var in other places, I think we can keep it they way it was.

Reply:
using below